stories/InputField.stories.js

```diff
@@ -57,7 +60,20 @@ function createStory(name, props) {
         ))
 
         .add('Status: Error', () => (
-            <InputField {...props} value="This value produces an error" error />
+            <>
+                <InputField
+                    {...props}
+                    value="This value produces an error"
+                    error
+                />
+                <Help error {...props}>
+                    This describes the error, if a message is supplied.
+                </Help>
+                <Help {...props}>
+                    This is some help text to advice what this input actually
+                    is.
+                </Help>
```

Comment:
Generally I would put the error text after the default help text to avoid more elements than necessary moving around.